UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/SalesCatalogue/ScsResponseQueueMessage.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace UKHO.ExchangeSetService.Common.Models.SalesCatalogue
+{
+    public class ScsResponseQueueMessage
+    {
+        public string BatchId { get; set; }
+        public long FileSize { get; set; }
+        public string ScsResponseUri { get; set; }
+    }
+}
```

Comment:
Could be SalesCatalogueServiceResponseQueueMessage

Reply:
Done